backend/app/cmd/server.go

```diff
@@ -637,6 +652,11 @@ func (a *serverApp) run(ctx context.Context) error {
 		log.Printf("[WARN] failed to resubmit comments with staging images, %s", e)
 	}
 
+	for _, siteID := range a.Sites {
```

Comment:
this unbounded concurrency makes me worry a little bit. In some extreme cases user may have a lot of sites

I suggest using SizedGroup here and in other places if you need to limit it with error handling - ErrSizedGroup from https://github.com/go-pkgz/syncs 

this package already in dependency and used by other parts of code to do a similar thing

Reply:
I've changed it to use `ErrSizedGroup`. It's global for all sites, and one worker processes one topic.